automata/regex/regex.py

```diff
@@ -33,7 +34,7 @@ def isequal(re1, re2, *, input_symbols=None):
     nfa1 = NFA.from_regex(re1, input_symbols=input_symbols)
     nfa2 = NFA.from_regex(re2, input_symbols=input_symbols)
 
-    return nfa1 == nfa2
+    return DFA.from_nfa(nfa1) == DFA.from_nfa(nfa2)
```

Comment:
@eliotwrobson I thought we got the NFA equality algorithm working correctly. Is that not actually the case (thus leading you to compare the DFAs instead)?

Reply:
Oh no, it was working fine but I was passing in the wrong NFA during testing and tried a bunch of random stuff before realizing it was my mistake 🤣 this got committed by accident. I think this change is reverted in the current version though.

Reply:
@eliotwrobson Gotcha. I can confirm it's reverted. Thanks for the clarity!